test/performance-charts/vitest.config.ts

```diff
@@ -3,16 +3,30 @@ import codspeedPlugin from '@codspeed/vitest-plugin';
 import react from '@vitejs/plugin-react';
 
 const isCI = process.env.CI === 'true';
+const isTrace = !isCI && process.env.TRACE === 'true';
 
 export default defineConfig({
   plugins: [...(isCI ? [codspeedPlugin()] : []), react()],
   test: {
     setupFiles: ['./setup.ts'],
-    environment: 'jsdom',
+    env: { TRACE: isTrace ? 'true' : 'false' },
+    environment: isTrace ? 'node' : 'jsdom',
```

Comment:
We should probably always run this in browser mode though

Reply:
Regardless if tracing or not I mean

Reply:
I didn't want to change the defaults. Using JSDOM is consistent with CI, so it's probably better to investigate performance issues. Once our CI is running in the browser, we can switch. What do you think?